packages/react-storage/src/components/StorageBrowser/actions/handlers/types.ts

```diff
@@ -37,3 +38,15 @@ export interface LocationData {
    */
   type: LocationType;
 }
+
+export interface LocationData {
+  bucket: string;
+  permission: Permission;
+  prefix: string;
+  type: LocationType;
+}
```

Comment:
Think this is already declared in the code base?

Reply:
oops merge weirdness